### PR TITLE
[ci] Refactor rspec tests

### PR DESCRIPTION
**Pull request by ChrisBr**
- Use new expect syntax instead of deprecated should
- Remove duplicated use_before_action shoulda-matchers

### CODE REVIEW

File: src/api/spec/controllers/webui/user_controller_spec.rb [hunk 1]
```diff
@@ -6,7 +6,6 @@
   let!(:admin_user) { create(:admin_user, login: "king") }
   let(:deleted_user) { create(:deleted_user) }
 
-  it { is_expected.to use_before_action(:require_login) }
```
Comment (mdeniz) [inline src/api/spec/controllers/webui/user_controller_spec.rb]: Why?

Comment (ChrisBr) [inline src/api/spec/controllers/webui/user_controller_spec.rb]: See line 10 ;)

Comment (mdeniz) [inline src/api/spec/controllers/webui/user_controller_spec.rb]: require_login and require_admin are two different before filters... they are defined in the user controller... I wanted to test it, what's wrong?

Comment (ChrisBr) [inline src/api/spec/controllers/webui/user_controller_spec.rb]: /me should wear glasses ... Sorry, my mistake, will fix it!

**Comment by mdeniz** [comment]
LGTM :+1:

**Pull request merged**